config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java

```diff
@@ -162,7 +167,7 @@ private void addClusterContent(ContainerCluster cluster, Element spec, ConfigMod
         addLegacyFilters(spec, cluster);  // TODO: Remove for Vespa 7
 
         // Athenz copper argos
-        addIdentity(spec, cluster, context.getDeployState().getProperties().configServerSpecs());
+        addIdentity(spec, cluster, context.getDeployState().getProperties());
```

Comment:
It's a bit ugly to propagate the full deploy properties. Please limit the scope of the arguments as much as possible.

Reply:
fixed